core/src/validator.rs

```diff
@@ -1012,6 +1013,28 @@ impl Validator {
             )),
         };
 
+        let vote_connection_cache = match vote_use_quic {
+            true => {
+                let vote_connection_cache = ConnectionCache::new_with_client_options(
+                    "connection_cache_vote_quic",
+                    1,
```

Comment:
would be nice to add a constant for this or even overload `DEFAULT_TPU_CONNECTION_POOL_SIZE`

Reply:
Done